collections/sentinel-2-l1c.yaml

```diff
@@ -27,7 +27,8 @@ Tags:
   - open data
   - sentinel
 License: |
-  [License](https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l1c/#attribution-and-use)
+  - <a href="https://www.sentinel-hub.com/tou/" target="_blank">Sentinel Hub Terms of Use</a>.
+  - <a href="https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l1c/#attribution-and-use" target="_blank">Attribution and Use </a>.
```

Comment:
Attribution and Use > Sentinel-2 L1C Attribution and Use

Reply:
done